pkg/option/config.go

```diff
@@ -2231,6 +2235,12 @@ type DaemonConfig struct {
 
 	// EnvoySecretNamespace for TLS secrets. Used by CiliumEnvoyConfig via SDS.
 	EnvoySecretNamespace string
+
+	// BPFMapEventBuffers has configuration on what BPF map event buffers to enabled
+	// and configuration options for those.
+	BPFMapEventBuffers          map[string]string
+	BPFMapEventBuffersValidator func(val string) (string, error) `json:"-"`
```

Comment:
Does this need to be exported?

Reply:
It's set in cmd/daemon_main.go, similar to the FixedIdentityMappingValidator (although it could be argued that neither of these are in the right place).